pt/lessons/libraries/benchee.md

```diff
@@ -310,22 +318,63 @@ Benchee.run(
 E quando rodamos agora como nosso formatador customizado, veremos:
 
 ```shell
-Operating System: macOS
-CPU Information: Intel(R) Core(TM) i5-4260U CPU @ 1.40GHz
-Number of Available Cores: 4
-Available memory: 8.589934592 GB
-Elixir 1.5.1
-Erlang 20.0
+Operating System: Linux
+CPU Information: Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz
+Number of Available Cores: 8
+Available memory: 15.61 GB
+Elixir 1.8.1
+Erlang 21.3.2
+
 Benchmark suite executing with the following configuration:
-warmup: 2.00 s
-time: 5.00 s
+warmup: 2 s
+time: 5 s
+memory time: 0 ns
 parallel: 1
 inputs: none specified
-Estimated total run time: 14.00 s
+Estimated total run time: 14 s
 
+Benchmarking flat_map...
+Benchmarking map.flatten...
+Average for flat_map: 419433.3593474056
+Average for map.flatten: 788524.9366408596
+```
+
+## Memória
+
+Estamos quase no fim, mas percorremos todo esse caminho sem mostrar a vocês uma das características mais legais do Benchee: medições de memória!
+
+Benchee é capaz de medir o consumo de memória, é limitado ao processo em que seu benchmark é executado. Ele não pode monitorar o consumo de memória em outros processos (como pools de workers).
+
+O consumo de memória inclui toda a memória que seu cenário de benchmarking usou - também a memória que foi coletada pelo garbage collector para que não represente necessariamente o tamanho máximo da memória do processo.
+
+Como você pode usá-lo? Bem, você acabou de usar a opção `:memory_time`!
+
+```
+Operating System: Linux
+CPU Information: Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz
+Number of Available Cores: 8
+Available memory: 15.61 GB
+Elixir 1.8.1
+Erlang 21.3.2
+
+Benchmark suite executing with the following configuration:
+warmup: 0 ns
+time: 0 ns
+memory time: 1 s
+parallel: 1
+inputs: none specified
+Estimated total run time: 2 s
 
 Benchmarking flat_map...
 Benchmarking map.flatten...
-Average for flat_map: 851.8840109326956
-Average for map.flatten: 1659.3854339873628
+
+Memory usage statistics:
+
+Name           Memory usage
+flat_map          624.97 KB
+map.flatten       781.25 KB - 1.25x memory usage +156.28 KB
+
+**All measurements for memory usage were the same**
 ```
+
+Como você pode ver, Benchee não se importa em exibir todas as estatísticas com todas as amostras, elas foram as mesmas. Isso é muito comum se suas funções não incluem uma quantidade de aleatoriedade. E de que serviriam todas as estatísticas se te exibissem sempre o mesmos valores?
```

Comment:
Just a thought here... "... estatísticas se _te_ exibissem ...". In this lesson it seems we use the third person when speaking to the interlocutor, maybe "_lhe_ exibissem" would be a better fit?

Also, a small typo here: "... exibissem sempre **o** mesmos valores ..." => os

Reply:
many thanks for revising my pull-request, you folks always doing the hardest parts <3